README.md

```diff
@@ -62,7 +62,13 @@ quickly get libdragon up and running. Basically:
 If you want, you can also compile and run one of the examples that will
 be found in `libdragon/examples` in the skeleton project.
 
-### Option 2: Compile the toolchain (Linux/macOS only)
+Note for Apple Silicon users: we ship and update the docker container image for both
+x86-64 and arm64, so this option works on Apple Silicon machines too.
```

Comment:
We still do it manually. Is GH actions apple silicon support near?

Reply:
Not that I know of. OTOH, we might be close to merge the patches to cross-compile the Windows toolchain from Linux, and maybe that will work also to cross-compile the M1 toolchain.